azure-pipelines.yml

```diff
@@ -38,4 +38,5 @@ steps:
       curl -sSL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s examples/mini-compile/src
       curl -sSL https://get.haskellstack.org/ | sh
       stack setup > /dev/null
+      stack --no-terminal build
```

Comment:
Would be nice to add a comment somewhere for why we need this despite calling the same command in `CI.hs` (once you’ve finished debugging).

Reply:
The next commit knocks that line out.

Reply:
For the record, that particular `stack exec` was working because the `stack build` that preceded it caused `extra` to be installed. That is, was serendipitous rather than good design.